src/ansys/aedt/core/mechanical.py

```diff
@@ -675,6 +675,50 @@ def assign_heat_generation(self, assignment, value, name=""):
             return bound
         return False
 
+    @pyaedt_function_handler()
+    def assign_2way_coupling(self, setup=None, number_of_iterations=2):
+        """Assign two-way coupling to a setup.
+
+        Parameters
+        ----------
+        setup : str, optional
+            Name of the setup. The default is ``None``, in which case the active setup is used.
+        number_of_iterations : int, optional
+            Number of iterations. The default is ``2``.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+
+        References
+        ----------
+
+        >>> oModule.AddTwoWayCoupling
+
+        Examples
+        --------
+
```

Comment:
@Samuelopez-ansys just for completeness I'd add imports, create instance and release desktop :)